### PR TITLE
city-guide: add yolanda's nyc entries

### PR DESCRIPTION
**Pull request by yolandaz**

### CODE REVIEW

**Comment by stephen** [comment]
do you want copy editing feedback or nah

File: city-guide/_cities/nyc/2021-10-patisserie-tomoko.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Patisserie Tomoko
+kind: dessert
+---
+Yolanda notes, "Another Japanese sweets cafe/bakery. We just got the ice cream sundae, which was delicious. They don't have any seating, so you have to just eat outside or nearby. Excited to try more of their stuff."
```
Comment (stephen) [inline city-guide/_cities/nyc/2021-10-patisserie-tomoko.md:5]: maybe note the relation to chaan?

**Comment by yolandaz** [comment]
yea i'll take copy editing feedback

File: city-guide/_cities/nyc/2021-10-prince-tea-house.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+title: Prince Tea House
+kind: dessert
+---
+Yolanda notes, "Came here a few times. Everything is pretty decent, but it's not one of my favorite dessert spots in NYC. It's easier to get seated than other dessert spots. Also, their afternoon tea was pretty cute."
```
Comment (stephen) [inline city-guide/_cities/nyc/2021-10-prince-tea-house.md:5]: consider staying present tense throughout the sentence?

File: city-guide/_cities/nyc/2021-10-patisserie-tomoko.md (hunk 1 above)
Comment (yolandaz) [reply city-guide/_cities/nyc/2021-10-patisserie-tomoko.md:5]: hm i looked it up and wasn't sure if they're actually related. something like one of the chefs left cha an and started this one, but they're not like sister restaurants or anything

Comment (stephen) [reply city-guide/_cities/nyc/2021-10-patisserie-tomoko.md:5]: ah ok. might be worth mentioning how similar the sundae is, but up to you

Comment (yolandaz) [reply city-guide/_cities/nyc/2021-10-patisserie-tomoko.md]: added!

**Pull request merged**